allennlp/modules/token_embedders/token_characters_encoder.py

```diff
@@ -37,10 +37,14 @@ def forward(self, token_characters: torch.Tensor) -> torch.Tensor:  # pylint: di
         return self._dropout(self._encoder(self._embedding(token_characters), mask))
 
     @overrides
-    def extend_vocab(self, extended_vocab: Vocabulary, vocab_namespace: str = "token_characters"):
+    def extend_vocab(self,  # pylint: disable=arguments-differ
+                     extended_vocab: Vocabulary,
+                     vocab_namespace: str = "token_characters",
+                     pretrained_file: str = None):
```

Comment:
Same two comments on this class (return type annotation and docstring wording tweak).